distribution/scripts/jmeter/apimchart.py

```diff
@@ -0,0 +1,164 @@
+# Copyright 2017 WSO2 Inc. (http://wso2.org)
```

Comment:
We might need to change the license header

Reply:
Updated